lib/engines/content_block_manager/app/models/content_block_manager/content_block/version.rb

```diff
@@ -10,6 +10,10 @@ class Version < ApplicationRecord
       def field_diffs
         self[:field_diffs] ? ContentBlock::DiffItem.from_hash(self[:field_diffs]) : {}
       end
+
```

Comment:
is this a stray method left over from a refactor?

Reply:
Whoops, yes!